opencv-examples/calibrate.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+import numpy as np
+import cv2
+import os
+import argparse
+import yaml
+import pickle
+from glob import glob
+
+# Taken from https://github.com/smidm/video2calibration and adopted to python 3
```

Comment:
We should probably also add the original copyright notice / license as a comment to this file:
https://github.com/smidm/video2calibration/blob/master/LICENSE.md